src/store/constants/index.js

```diff
@@ -1,2 +1,6 @@
 // eslint-disable-next-line
```

Comment:
Good Job Dan,
However, this (`// eslint-disable-next-line`) is not required as the code below did not violate the project's coding standards. Please let's remove it and other similar cases